### ISSUE TITLE
Championship Podiums display

### ISSUE THREAD

**Issue by Mollerz** [thewca/wca-live#36]
After the UK Championship this weekend, before we started announcements we had to write out a list of who was on each UK podium manually which was slightly annoying.

It would be nice for Admins to have a classification setting, which shows normal podiums, championship podiums, possibly a list of any NRs, CRs, WRs achieved during the competition as well. Ideally it would be clean, only show the podiums + winning result for each person.

**Comment by jonatanklosko** [comment]
Duplicate of #16, but as this is a bit more elaborated I'm gonna close that one.

**Comment by Mollerz** [comment]
Bumping this as we had to do it again and some of our staff made a couple of mistakes meaning we missed OH by accident in the announcements and also accidentally started doing 7x7 based on first round results since we were rushing to get them written out.

**Comment by Goosly** [comment]
@Mollerz  As a workaround, you can use the certificates tool to generate certificates for the UK winners and then you have a PDF you can read from, or write down a list from, rather than figuring out the list manually.
Obviously it would be nice to have this in WCA Live too, just letting you know about this workaround that others are using :-)
